website/project/model.py

```diff
@@ -730,6 +731,23 @@ def category_display(self):
         """The human-readable representation of this node's category."""
         return self.CATEGORY_MAP[self.category]
 
+    # We need the following 2 properties in order to serialize related links in NodeRegistrationSerializer
+    @property
+    def registered_user_id(self):
```

Comment:
Not too familiar with ODM but here's where I would change the way that DRF handles the lookup_field if we were using the Django ORM. https://github.com/tomchristie/django-rest-framework/blob/3.1.1/rest_framework/relations.py#L180-L189 I believe you could just override get_object and you could make lookup_field deal with 'something.something'